Multisig/UI/Transactions/TransactionsList/TransactionsViewModel.swift

```diff
@@ -57,10 +53,13 @@ class TransactionsViewModel: LoadableViewModel {
             .sink(receiveCompletion: { completion in
                 if case .failure(let error) = completion {
                     self.errorMessage = error.localizedDescription
+                    App.shared.viewState.show(message: error.localizedDescription)
```

Comment:
App.shared.viewState can be defined as private parameter

Reply:
I think this way it is more explicit that we work with a global state